modules/common/condition/conditions.go

```diff
@@ -328,6 +328,21 @@ const (
 	// RabbitMqTransportURLReadyErrorMessage
 	RabbitMqTransportURLReadyErrorMessage = "RabbitMqTransportURL error occured %s"
 
+	//
+	// NotificationBusInstanceReady condition messages
+	//
+	// NotificationBusInstanceReadyInitMessage
+	NotificationBusInstanceReadyInitMessage = "NotificationBusInstance not started"
```

Comment:
should we also have a dedicated condition for it, like the `RabbitMqTransportURLReadyCondition` ?

Reply:
makes sense, I think I missed that one! good catch. Adding in my next PS.